saleor/webhook/transport/synchronous/circuit_breaker/breaker_board.py

```diff
@@ -50,21 +58,24 @@ def register_success(self, app_id: int):
 
     def __call__(self, func):
         def inner(*args, **kwargs):
-            delivery: EventDelivery = args[0]
-            if delivery.event_type not in self.webhook_event_types:
+            event_type: str = args[0]
+            webhook: Webhook = args[2]
+
+            if event_type not in self.webhook_event_types:
                 return func(*args, **kwargs)
 
             # TODO - ensure webhook and app are preloaded
-            app = delivery.webhook.app
+            app = webhook.app
 
             if not self.is_closed(app.id):
                 return WebhookResponse(content=""), None
 
-            response, data = func(*args, **kwargs)
-            if response.status == EventDeliveryStatus.FAILED:
+            response = func(*args, **kwargs)
+            if response is None:
```

Comment:
We need to be absolutely sure that successful webhook executions never return `None` as a response.

Reply:
yes, webhook execution will return `None` only if there is an actual failure, in case of success, even with empty data, it will either be `[]` or `{}`, thus not explicitly `None`.